Formula/infer.rb

```diff
@@ -127,8 +95,30 @@ def install
       }
     EOS
 
-    shell_output("#{bin}/infer --fail-on-issue -P -- clang -c FailingTest.c", 2)
-    shell_output("#{bin}/infer --fail-on-issue -P -- clang -c PassingTest.c")
+    no_issues_output =
+      ["", "  No issues found  ", ""].join("\n")
+
+    failing_c_output =
+      ["", "FailingTest.c:5: error: Null Dereference",
+       "  pointer `s` last assigned on line 4 could be null and is dereferenced at line 5, column 3.",
+       "  3. int main() {",
+       "  4.   int *s = NULL;",
+       "  5.   *s = 42;",
+       "       ^",
+       "  6. ",
+       "  7.   return 0;",
+       "",
+       "",
+       "Found 1 issue",
+       "          Issue Type(ISSUED_TYPE_ID): #",
+       "  Null Dereference(NULL_DEREFERENCE): 1",
+       ""].join("\n")
```

Comment:
```suggestion
    failing_c_output = <<~EOS

      FailingTest.c:5: error: Null Dereference
        pointer `s` last assigned on line 4 could be null and is dereferenced at line 5, column 3.
        3. int main() {
        4.   int *s = NULL;
        5.   *s = 42;
             ^
        6.
        7.   return 0;
      
      
      Found 1 issue
                Issue Type(ISSUED_TYPE_ID): #
        Null Dereference(NULL_DEREFERENCE): 1

    EOS
```
Formatting might need work.

Reply:
Okay, so I did this (manually) and now the tests pass, but `brew style` complains because of the formatting.

Any suggestions here?

Reply:
Indent the heredoc as suggested, and drop the whitespace inside `main` in `FailingTest.c` so that we don't need `6. ` with the trailing whitespace.